src/main/java/de/cismet/cids/custom/wunda_blau/search/server/QsgebStatusLightweightSearch.java

```diff
@@ -0,0 +1,241 @@
+package de.cismet.cids.custom.wunda_blau.search.server;
+
+import Sirius.server.middleware.interfaces.domainserver.MetaService;
+import Sirius.server.middleware.types.MetaClass;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import org.apache.log4j.Logger;
+
+import org.openide.util.lookup.ServiceProvider;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+
+import de.cismet.cids.dynamics.CidsBean;
+
+import de.cismet.cids.server.search.AbstractCidsServerSearch;
+import de.cismet.cids.server.search.SearchException;
+
+import de.cismet.cidsx.base.types.Type;
+
+import de.cismet.cidsx.server.api.types.SearchInfo;
+import de.cismet.cidsx.server.api.types.SearchParameterInfo;
+import de.cismet.cidsx.server.search.RestApiCidsServerSearch;
+import de.cismet.cidsx.server.search.builtin.legacy.LightweightMetaObjectsSearch;
+
+import de.cismet.connectioncontext.ConnectionContext;
+import de.cismet.connectioncontext.ConnectionContextStore;
+
+/**
+ * Builtin Legacy Search to delegate the operation getLightweightMetaObjectsByQuery to the cids Pure REST Search API.
+ *
+ * @author   sandra
+ * @version  $1.0$, $31.05.2018$
+ */
+@ServiceProvider(service = RestApiCidsServerSearch.class)
+public class QsgebStatusLightweightSearch extends AbstractCidsServerSearch implements RestApiCidsServerSearch,
+    LightweightMetaObjectsSearch,
+    ConnectionContextStore {
+
+    //~ Static fields/initializers ---------------------------------------------
+
+    private static final Logger LOG = Logger.getLogger(QsgebStatusLightweightSearch.class);
+
+    private static final String TABLE_QSGEB_STATUS = "qsgeb_status"; 
+    public static final String FIELD__STATUS_ID = "status.id";
+
+    //~ Enums ------------------------------------------------------------------
+
+    /**
+     * DOCUMENT ME!
+     *
+     * @version  $Revision$, $Date$
+     */
+    public enum SearchFor {
+
+        //~ Enum constants -----------------------------------------------------
+
+        STATUS
+    }
+
+    //~ Instance fields --------------------------------------------------------
+
+    private ConnectionContext connectionContext = ConnectionContext.createDummy();
+
+    @Getter private final SearchInfo searchInfo;
+    @Getter @Setter private SearchFor searchFor;
+    @Getter @Setter private Integer statusId;
+    @Getter @Setter private String representationPattern;
+    @Getter @Setter private String[] representationFields;
+
+    //~ Constructors -----------------------------------------------------------
+
+    /**
+     * Creates a new LightweightMetaObjectsByQuerySearch object.
+     */
+    public QsgebStatusLightweightSearch() {
+        this.searchInfo = new SearchInfo(
+                this.getClass().getName(),
+                this.getClass().getSimpleName(),
+                "Builtin Legacy Search to delegate the operation getLightweightMetaObjectsByQuery to the cids Pure REST Search API.",
```

Comment:
Wrong description of this search (was probably copied and pasted from another search). Please add a specific description for this search.